samwise.js

```diff
@@ -5,36 +5,38 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!